### ISSUE TITLE
Improve navigation by making translations clickable

### ISSUE THREAD

**Issue by adamzkover** [hunnor-dict/web-spring#4]
Each translation is in a separate element, e.g. `<span class="trans">lastebil</span>`.
Clicking on a span should open lookup of that translation, in this case navigate to /?term=lastebil
Use JavaScript to create this functionality and add some subtle styling with CSS.

**Comment by Alessandro-Battiato** [comment]
Greetings!
I can see that your project mainly uses Java, but for your issue only JavaScript and Css are needed am I right? If you can confirm this then could you assign this issue to me? :)

**Comment by adamzkover** [comment]
Yes, that is correct, JavaScript and CSS should be enough to solve this issue. Thank you for being interested in looking into this! :)

**Comment by Alessandro-Battiato** [comment]
I just forked your repository and I have two questions, sorry for my lack of familiarity with everything but I'm not that experienced in working on projects of other people, especially with this one that needs Maven :)
- 1 I read some articles online trying to understand how to use maven, but I'm a bit confused, when using it am I going to create a new jar file everytime and then run the command java -jar on the newly created file? Or is the process much simpler as you described in your readme file and I'm just overthinking it? Also regarding maven's website https://maven.apache.org/download.cgi , is there no difference between the binary and source files when I download the program?
- 2 I could already give a look to the website using the link provided in the readme file, and I can see the issue very clearly, maybe I'm missing it but what's exactly the API you're using? If I could have this information and understand a bit more how the project works and how to use maven I could immediately start implementing the functionality of redirecting the user to the proper translation's page when clicking on a span

**Comment by adamzkover** [comment]
1 Yes, when you do modifications you can use `mvn clean package` to make a new JAR file and then run it with `java -jar`. You should download the binary of Maven and install it according to the instructions for your platform. Make sure you have Java 11 or a later version.

2 The backend is a Solr server with which the Java code interacts. You would probably need that as well for development if you want a complete local copy of the system.

If you are not familiar with Maven, Solr and Docker, you could probably just download a complete copy of the webpage that has a search, e.g. https://dict.hunnor.net/?term=%C3%A1fonya and then take out reference to the minified JavaScript, which is now https://dict.hunnor.net/js/hunnor-web.min-27bc2af8fa20c29c4ba23e6f3ebfbd1f.js and replace it with your modified JavaScript code (using https://github.com/hunnor-dict/web-spring/blob/master/src/main/web/js/search.js as a starting point). 

Great to have you contributing! Let me know if you have more questions.

**Comment by Alessandro-Battiato** [comment]
I want to thank you once again for your availability and for answering all my questions, but I have to deeply apologize for wasting your time as I've realized after thoroughly analyzing the project and the tools used that this is out of my capabilities, it's sadly out of my reach and it would take me too much time to understand how to work on this project and I already have other responsibilities at the moment, but thanks anyway for the help and I hope you'll find soon enough the right contributer :)

**Comment by adamzkover** [comment]
Sure, no problem! The effort is not lost, the notes will be here for future contributors (maybe even my future self ;)
